frontend/.env.example

```
@@ -2,4 +2,5 @@
 VITE_API_BASE_URL=http://localhost:3000
 VITE_APP_NAME=Tazama Case Management System
 VITE_APP_VERSION=0.0.1
-VITE_CRYPTO_KEY=rCRrt08XYmYZrUoKffOQWXa327AJyBxv
\ No newline at end of file
+VITE_CRYPTO_KEY=rCRrt08XYmYZrUoKffOQWXa327AJyBxv
+VOILA_URL=http://10.10.80.20:8866
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)